bigquery/datatransfer/cloud-client/README.md

```diff
@@ -0,0 +1,29 @@
+# Getting Started with BigQuery Data Transfer API
+
+[BigQuery Data Transfer Service][BigQuery Data Transfer] features an API that
+allows developers to create transfer jobs from data sources to BigQuery.
+These sample Java applications demonstrate how to access the BigQuery Data
+Transfer API using the [Google Cloud Client Library for
+Java][google-cloud-java].
+
+[BigQuery Data Transfer]: https://cloud.google.com/bigquery/docs/transfer-service-overview
+[google-cloud-java]: https://github.com/GoogleCloudPlatform/google-cloud-java
+
```

Comment:
There should be a link at least to something about Auth?

Reply:
Done. Added a note about auth to both the sample and this readme (in the Quickstart section).

Since auth is needed across products, I think we should investigate using [Jason's README generator](https://github.com/GoogleCloudPlatform/nodejs-repo-tools). I believe Python, Ruby, and Node.js are all using it already.